.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: Run CI
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
doesn't this also require `tags: ['*']`? Given our usual git tag push -> PyPI release flow